doc/manual/architecture.md

```diff
@@ -32,9 +32,11 @@ The following requirement list was the foundation for the design of the framewor
     simulation. These are "driven" in dynamic tests.
   - OpenSCENARIO (XOSC): Scenario definition, includes route definition (one file per route with
     extension .xosc)
-- **Check/Checker:** A software module or routine that checks exactly one rule
-  or creates statistical information from the route model. This can be a static
-  or a dynamic test. For a list of possible tests, see parent page.
+- **Rule:** A textual description/definition about the input file
+  characteristics.
+- **Check/Checker:** A software module that addresses to check one or multiple
+  rules or creates statistical information from the input file model. This can
+  be a static or a dynamic test. For a list of possible tests, see parent page.
```

Comment:
```suggestion
- **Check/Checker:** A software module that tests rules compliance
  or creates statistical information about the input file. This can
  be a static or a dynamic test. For a list of possible tests, see parent page.
```